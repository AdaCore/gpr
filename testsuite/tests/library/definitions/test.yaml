description:
    check for library definitions
driver: build_and_run
project_file: lib_def.gpr
main: main
output:
- ["test.out.win", "'windows' in env.host.platform"]
