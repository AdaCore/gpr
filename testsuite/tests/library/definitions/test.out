Demo     K_LIBRARY
Library_Name : abc
Library_Directory : lib
                  : lib/
Library Filename : libabc.so.1.2
                 : lib/libabc.so.1.2
Library Version Filename : libabc.so.1.2
                         : lib/libabc.so.1.2
Library Major Version Filename : libabc.so.1
                               : lib/libabc.so.1
Library_Standalone : NO
Demo     K_LIBRARY
Library_Name : abc
Library_Directory : lib
                  : lib/
Library Filename : libabc.so.1.4
                 : lib/libabc.so.1.4
Library Version Filename : libabc.so.1.4
                         : lib/libabc.so.1.4
Library Major Version Filename : libabc.so.1
                               : lib/libabc.so.1
Library_Standalone : NO
Demo     K_LIBRARY
Library_Name : abc
Library_Directory : release
                  : lib/release/
Library Filename : libabc.so.1.4
                 : lib/release/libabc.so.1.4
Library Version Filename : libabc.so.1.4
                         : lib/release/libabc.so.1.4
Library Major Version Filename : libabc.so.1
                               : lib/release/libabc.so.1
Library_Standalone : NO
Demo     K_LIBRARY
Library_Name : abc
Library_Directory : release
                  : lib/release/
Library Filename : libabc.so.A.B
                 : lib/release/libabc.so.A.B
Library Version Filename : libabc.so.A.B
                         : lib/release/libabc.so.A.B
Library Major Version Filename : libabc.so.A
                               : lib/release/libabc.so.A
Library_Standalone : NO
